engines/account/app/mailers/account/signup_mailer.rb

```diff
@@ -0,0 +1,20 @@
+module Account
+  class SignupMailer < ActionMailer::Base
+
+    default from: 'info@katuma.org'
+
+    # @param signup [Account::Signup]
+    def confirm_email(signup)
+      @signup = signup
+      @url = confirm_url
+
+      mail to: @signup.email, subject: "Welcome to Katuma!"
```

Comment:
yep that's something that we need ASAP of course, maybe last thing after all the behaviour works as expected